Gemfile

```diff
@@ -66,6 +66,8 @@ group :test do
   gem 'cuprite', '~> 0.14'
   gem 'erb_lint', '~> 0.4', require: false
   gem 'factory_bot_rails', '~> 6.2'
+  # Required for rspec-retry for some specs post Faraday 2.0
```

Comment:
Nit: Not a blocker, but would the commit message provide enough context for why this is needed? 
I've been in way too many too many Gemfiles dotted with out of date comments 😆

Reply:
I was 50/50 on adding it in the first place so happy to take it out. Any objection to removing the `use redis for action cable` comment while I'm there for a completely :sparkles: clean :sparkles: Gemfile?

Reply:
aha good spot, no objections 💥